common/transport/http/package.json

```diff
@@ -19,8 +19,9 @@
     "sinon": "^7.4.1",
     "source-map-support": "^0.5.16",
     "ts-node": "^8.6.2",
-    "tslint": "^6.1.3",
-    "typescript": "3.7.5"
+    "tslint": "^6.1.0",
```

Comment:
Why did we go backwards in the version of tslint?

Reply:
There is no reason :) Rolled back